### DISCUSSION TITLE
why strongswan package not the same version on centos and ubunto?

### DISCUSSION THREAD

**Discussion by realbitt** [strongswan/strongswan#1634] [General]
I have centos 8 stream I notice that the version is updated continuously and quickly, while on Ubuntu, until this moment, the available version is 5.8.2?

is there any way to install 5.9.9 on Ubuntu?

**Comment by tobiasbrunner** [comment]
> I have centos 8 stream I notice that the version is updated continuously and quickly, while on Ubuntu, until this moment, the available version is 5.8.2?

Different distributions have different release models and cycles. Debian/Ubuntu freezes packages versions pretty much for their major releases (with some exceptions like browsers).

> is there any way to install 5.9.9 on Ubuntu?

[From sources](https://docs.strongswan.org/docs/5.9/install/install.html#_compile_strongswan_yourself), make sure to remove any installed strongSwan-related packages before you do that. (You could technically also build your own .deb packages).

Note that 5.9.9 is not the latest release.
